scripts/lib/website-comparisons.ts

```diff
@@ -33,17 +33,19 @@ export function isPdf(link: string, response: string): boolean {
   return false;
 }
 
-// Given a website link and its html, return a hash of the cleaned website. Do not return a hash if the site is for a PDF.
+// Given a website link and its data, return a hash of the cleaned website. If it's a PDF, attempt to convert to readable text first.
```

Comment:
This comment is a bit confusing because if it's a PDF, it's *already* been converted to readable text.